site/assets/scss/_navbar.scss

```diff
@@ -75,17 +84,81 @@
   }
 
   .dropdown-menu {
-    --#{$prefix}dropdown-min-width: 12rem;
-    --#{$prefix}dropdown-link-hover-bg: rgba(var(--bd-violet-rgb), .1);
-    @include rfs(.875rem, --#{$prefix}dropdown-font-size);
+    --bs-dropdown-min-width: 12rem;
+    --bs-dropdown-padding-x: .25rem;
+    --bs-dropdown-padding-y: .25rem;
+    --bs-dropdown-link-hover-bg: rgba(var(--bd-violet-rgb), .1);
+    --bs-dropdown-link-active-bg: rgba(var(--bd-violet-rgb), 1);
+    @include rfs(.875rem, --bs-dropdown-font-size);
+    @include font-size(.875rem);
+    @include border-radius(.5rem);
     box-shadow: $dropdown-box-shadow;
+
+    li + li {
+      margin-top: .125rem;
+    }
+
+    .dropdown-item {
+      @include border-radius(.25rem);
+
+      &:active {
+        .bi {
+          color: inherit !important; // stylelint-disable-line declaration-no-important
+        }
+      }
+    }
+
+    .active {
+      font-weight: 600;
+
+      .bi {
+        display: block !important; // stylelint-disable-line declaration-no-important
+      }
+    }
+  }
+}
+
+[data-bs-theme="dark"] {
```

Comment:
The issue is not really here at this line, but in dark mode, the contrast ratio doesn't seem correct 

![Screenshot 2022-11-24 at 17 39 40](https://user-images.githubusercontent.com/17381666/203833846-06d3babd-b4ee-4eb3-a409-febd9279aa9f.png)


![Screenshot 2022-11-24 at 17 44 50](https://user-images.githubusercontent.com/17381666/203833862-5cef61a2-e862-420e-beba-85239bd2e4f1.png)

If the values found with the color picker are correct: foreground `#f0e9ff` and background `#9461fb`